plugin/src/templates/utils.ts

```diff
@@ -43,6 +97,28 @@ export function prepareFilesystem(cacheDir: string): void {
   // Gatsby uses this instead of fs if present
   // eslint-disable-next-line no-underscore-dangle
   global._fsWrapper = lfs
+
+  console.log('Starting to prepare data directory')
+
+  if (
+    process.env.GATSBY_EXCLUDE_DATASTORE_FROM_BUNDLE === 'true' ||
+    process.env.GATSBY_EXCLUDE_DATASTORE_FROM_BUNDLE === '1'
+  ) {
```

Comment:
tl;dr - Opted to hard code this rather than use `shouldSkipBundlingDatastore` because of import issues that result when the lambda functions are compiled.

_Longer version_
Initially I had changed this to use the `shouldSkipBundlingDatastore` method that's in the `helpers/config` file. While locally this works, when the functions are built, an import error results because `helpers/config` is not included in the lambda bundle.

We could have included the file in the lambda bundle so that the lambda had a reference to `shouldSkipBundlingDatastore`, but that didn't make the most sense given that the file mostly contains logic related to the build plugin and the configuration files that should be generated as part of the build process.

We could have also moved the `shouldSkipBundlingDatastore` and `isEnvSet` methods (on which `shouldSkipBundlingDatastore` depends on) into the `templates/utils` file, but that also didn't make the most sense since these are configuration related details, not related to templating.

Though not perfect, for the time being this seemed like the simplest/most straightforward solution.